tests/unittests/test_sync_batches_retry.py

```diff
@@ -0,0 +1,154 @@
+import unittest
```

Comment:
@savan-chovatiya What is the difference between the test cases written in this file and `test_attribute_error_retry.py` file.
In the above file as well, you are checking that call_count should be 5 times. What else you are checking in `test_sync_batches_retry` file?

Reply:
There was no backoff on the `sync_batches` function so I added and also include the `FacebookRequestError` error with `AttributeError` error so those tests are under `test_sync_batches_retry`. While other test cases for `AttributeError` are under 'test_attribute_error_retry.py'